src/bcf/report/table_report/report_table.html.tera

```diff
@@ -10,6 +10,7 @@
 <body>
 <script src="../../js/jquery.min.js"></script>
 <script src="../../js/bootstrap-table.min.js"></script>
+<script src="../../js/bootstrap.bundle.min.js"></script>
```

Comment:
This line needs to be included in the expected files [here](https://github.com/rust-bio/rust-bio-tools/blob/master/tests/expected/report/details/a/KRAS.html) and [here](https://github.com/rust-bio/rust-bio-tools/blob/master/tests/expected/report/details/b/KRAS.html). Also the `sed` command need to be changed: https://github.com/rust-bio/rust-bio-tools/blob/832bf69e8fff957d334f54984c02da06d4557f73/tests/lib.rs#L208
Since only one line is added at the beginning it should be ok to change the command to '31d;24d'.